### ISSUE TITLE
[Bug] [web] web ui cannot be displayed

### ISSUE THREAD

**Issue by impactCn** [apache/seatunnel#6074]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

![image](https://github.com/apache/seatunnel/assets/35489877/2777c854-4c09-472e-9cdf-3cfa062d3220)
I checked that the seatunnel service and web service started normally.
![image](https://github.com/apache/seatunnel/assets/35489877/1ee41be5-3a6f-481e-b5b5-45454ae5a5c1)
![image](https://github.com/apache/seatunnel/assets/35489877/5cc72ea3-4757-4af3-ac29-a6690b1f758b)

### SeaTunnel Version

2.3.3

### SeaTunnel Config

```conf
server:
  port: 8801

spring:
  application:
    name: seatunnel
  jackson:
    date-format: yyyy-MM-dd HH:mm:ss
  datasource:
    driver-class-name: com.mysql.jdbc.Driver
    url: jdbc:mysql://127.0.0.1:3306/seatunnel?useSSL=false&useUnicode=true&characterEncoding=utf-8&allowMultiQueries=true&allowPublicKeyRetrieval=true
    username: root
    password: 123456
  mvc:
    pathmatch:
      matching-strategy: ant_path_matcher

jwt:
  expireTime: 86400
  secretKey: https://github.com/apache/seatunnel
  algorithm: HS256
```

### Running Command

```shell
sh seatunnel-backend-daemon.sh start
```

### Error Exception

```log
{"timestamp":"2023-12-25 07:04:23","status":404,"error":"Not Found","path":"/ui/index.html"}
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by Hu-WF** [comment]
Do not execute scripts in the bin directory of web ui，Just run：
`sh ./bin/seatunnel-backend-daemon.sh start
`

**Comment by impactCn** [comment]
> Do not execute scripts in the bin directory of web ui，Just run： `sh ./bin/seatunnel-backend-daemon.sh start `

ok, let me try again. thank

**Comment by feiyuthomas** [comment]
启动前端     不要再bin目录下启动  
sh ./bin/seatunnel-backend-daemon.sh start
